shared-libs/search/src/search.js

```diff
@@ -40,7 +40,11 @@ module.exports = function(Promise, DB) {
     intersection = _.uniq(intersection, 'id');
     _.each(responses, function(response) {
       intersection = _.reject(intersection, function(row) {
-        return !_.findWhere(response, { id: row.id });
+        const existent = _.findWhere(response, { id: row.id });
+        if (existent && 'sort' in existent) {
```

Comment:
This is a bit strange to mutate the row object inside a `_.reject` block.

Reply:
I changed it to a `.filter` function, but still mutating. I'd really rather not save the values separately and iterated _again_.  :(